src/doc/HOWTO/EMUfailure.sgml

```diff
@@ -358,7 +358,63 @@ the startup screen it runs fine though.
 </listitem>
 </itemizedlist>
 </sect2>
+</sect1>
 
+<sect1>
+<title>Differences in behaviour between Dosemu and Dosemu2</title>
+<para>
+The following differences may be apparent if you have previously been
+using Dosemu.
+</para>
+<sect2>
+<title>
+Filesystems
+</title>
+
+<sect3>
+<title>
+MFS
+</title>
+<para>
+Network device DOS filesystem that provides read/write access to the host filesystem.
+</para>
+<itemizedlist>
+<listitem>
+<para>
+Dosemu2 does not redirect a drive to arbitrary host paths on the command
+line of the emufs.sys driver when it is loaded in config.sys. Use the lredir2
+command to accomplish this in autoexec.bat instead.
+</para>
+</listitem>
+<listitem>
+<para>
+Dosemu2 does not support SUBST / JOIN when using its long filename support
```

Comment:
This is too vague, as it sounds as if
subst/join are completely unavailable
when you just enable lfn support.

Reply:
Okay will update.